content/applications/marketing/marketing_automation/advanced/understanding_metrics.rst

```diff
@@ -1,80 +1,173 @@
+
+Activity filter tab
+-------------------
+
+Next to the :guilabel:`graph` tab on the activity block, there's the option to open a
+:guilabel:`filter` tab (represented by a :guilabel:`filter/funnel` icon).
+
+.. image:: understanding_metrics/activity-filter-tab.png
    :align: center
-   :alt: Lead generation filters used to refine records on marketing automation campaigns in Odoo.
+   :alt: What a campaign activity filter tab looks like in Odoo Marketing Automation.
 
-At the top of the marketing automation form, there are a series of smart buttons. The
-:guilabel:`Participations` smart button in this example indicates that, out of those 18 records, 25
-have become participants. In other words, they have matched the criteria, and have already
-interacted with the campaign in one way or another.
+Clicking the :guilabel:`filter` tab on an activity block, reveals what the specific filters are for
+that particular campaign activity, and how many records in the database match that specific
+criteria.
 
-.. note::
-   :guilabel:`Records` represent a real-time number. Therefore, while the workflow is running,
-   changes in opportunity records (deletions, additions, adjustments) may occur. This means there
-   will be real-time updates to the number of :guilabel:`Records`, but such updates do *not* change
-   the number of :guilabel:`Participants`, as the metric *will not* exclude opportunities that have
-   already been set as :guilabel:`Participants` before - Odoo will just add new ones. That's why
-   the number of :guilabel:`Records` can be different from the number of :guilabel:`Participants`
-   from time to time.
-
-.. image:: understanding_metrics/campaign-records-vs-participants.png
+.. tip::
+   Clicking on the :guilabel:`records` link beneath the displayed filter reveals a separate pop-up
+   window containing a list of all the records that match that specific campaign activity rule(s).
+
+Link tracker
+============
+
+Odoo tracks all URLs used in marketing campaigns. To access and analyze those URLs, navigate to
+:menuselection:`Marketing Automation app --> Reporting --> Link Tracker`. Doing so reveals a
+:guilabel:`Link Statistics` page, wherein all campaign-related URLs can be analyzed.
+
+.. image:: understanding_metrics/campaign-link-tracker.png
    :align: center
-   :alt: Difference between real-time record count and total participants on a markeing campaign.
+   :alt: What a campaign activity filter tab looks like in Odoo Marketing Automation.
 
-Whenever the system runs, updating numbers and triggering actions, it will look at the
-:guilabel:`Target` model, and check if new :guilabel:`Records` have been added (or modified),
-which always keeps the flow up-to-date.
+The default view on the :guilabel:`Link Statistics` page is the :guilabel:`Bar Chart` view, but
+there are different view options available in the upper-left corner. There is the option to view the
+statistics as a :guilabel:`Line Chart` or :guilabel:`Pie Chart`.
```

Comment:
Although it's useful to know how the data is displayed, it's also useful to know *what* data is being displayed.
For instance, why is it important for marketers to know which links their prospective customers clicked on to see the marketing campaign?

Reply:
I totally understand this suggestion - completely get where you're coming from. However, in the past, I've been instructed to stick to "how" to access data -- and avoid explaining "why" certain data is useful -- because the user will determine their own value to the data presented to them, if that makes sense. So, with that in mind, I think I'll keep this as-is, but if it comes up in another review, I'll add it in. Thanks! 👍